cluster/external/provider.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function _kubectl() {
+    kubectl "$@"
+}
+
+# The external cluster is assumed to be up.  Do a simple check
+function up() {
+    _kubectl version >/dev/null
+    if [ $? -ne 0 ]; then
+        echo -e "\n*** Unable to reach external cluster.  Please check configuration ***"
+        echo -e "*** Type \"kubectl config view\" for current settings               ***\n"
+        exit 1
+    fi
+    echo "Cluster is up"
```

Comment:
Hard coding the image pull policy here isn't right IMHO.  If you apply my other pull request (https://github.com/kubevirt/kubevirt/pull/1352)
none of this stuff is necessary.

Reply:
The wole purpose of the providers is that you can run `make cluster-up` once and that then subsequent repreted calls to `make cluster-sync` synchronize the cluster with your code changes.

If you don't hardcode `Always` here, then you would only get changes synchronized to the nodes on the first invocation of `make cluster-sync`. On the second `make cluster-sync` invocation, you would only push the images to the registry but the newly created manifests would never pull the new manifests to the nodes.

If you only want to build the code and once push it to an external cluster then I would not use providers. It is really only for the developer flow.

Reply:
My point is that image pull policy is now mutable by an environment variable (IMAGE_PULL_POLICY) so it doesn't make sense to hard code it.

Reply:
I understand that. I tried to explain why it is important to explicitly hardcode it in this case. There is no other reasonable value than `Always` in the context of this provider. Otherwise, if you ever forget to set it to `Always`, you will have awkward results when running `make cluster-sync`.